libyul/backends/evm/StackHelpers.h

```diff
@@ -254,6 +254,18 @@ class Shuffler
 					ops.isCompatible(sourceTop, offset) // The lower position wants to have this slot.
 				)
 				{
+					// We cannot swap that deep.
+					if (ops.sourceSize() - offset - 1 > 16)
+					{
+						// If there is a reachable slot to be removed, park the current top there.
+						for (size_t swapDepth: ranges::views::iota(1u, 16u) | ranges::views::reverse)
+							if (ops.sourceMultiplicity(ops.sourceSize() - 1 - swapDepth) < 0)
```

Comment:
Can this underflow?

Reply:
Nope, since ``ops.sourceSize() - 1 > 16`` and ``swapDepth < 16``. But actually, I can let ``swapDepth`` range until *including* 16.

Reply:
Just increased it the iota range by one. Could be the last slot saving us.